lib/pf/web/constants.pm

```diff
@@ -233,7 +233,9 @@ Return a regex that would match all the portal profile uri: filter
 =cut
 
 sub _clean_urls_match_filter {
-    return @uri_filters;
+    local $_;
+    # REMOVE ME !!!!!! TEMP FIX WHILE JAMES FIXES THIS
```

Comment:
Is it fixed ?
If not, is there an issue entry cause, "once merged, who knows what will happen with this"